packages/api/models/schemas.js

```diff
@@ -400,16 +399,14 @@ module.exports.granule = {
     timestamp: { type: 'integer' },
     createdAt: { type: 'integer' },
     updatedAt: { type: 'integer' },
-    dataType: { type: 'string' },
```

Comment:
These values didn't get moved forward to the P3 lib/schemas.js and should be.    We probably should just do a quick audit to make sure prior merges didn't miss this somehow.

Reply:
It looks like they did? https://github.com/nasa/cumulus/blob/feature/rds-phase-3/packages/api/lib/schemas.js#L366

I don't think I'm following. You're thinking that the dataType/version should be... removed... from api/lib/schemas.js? Because they're not represented in PG maybe?

Reply:
@npauzenga  yes, that's right, the schemas.js in the lib should match the incoming changes from the model schema in main.

Reply:
Gotchya. We made updates in main to the _model_ schema and, in the feature branch switched to the _lib_ schema.

https://github.com/nasa/cumulus/pull/3181/commits/d40def2238dea8ab67bfc510678718cc8bbbe522